README.md

```diff
@@ -4,7 +4,8 @@ To start the app:
 
   * Install dependencies with `mix deps.get`
   * Create and load your database with `mix ecto.create && mix ecto.load`
-  * Install Node.js dependencies with `yarn install`
+  * Install Node.js dependencies with `cd assets && yarn && cd ..`
+  * Configure your development secrets `cp config/dev.secret.example.exs config/dev.example.exs` instructions for your OAuth setup can be found [here](https://developers.google.com/identity/sign-in/web/devconsole-project)
```

Comment:
May be also add a sentence about registering on http://cloudinary.com and adding the API key and secret in the `dev.secrets.exs`?